src/lib/plugin_apis/fs.api

```diff
@@ -2698,4 +2698,92 @@ gboolean bd_fs_udf_check_uuid (const gchar *uuid, GError **error);
  */
 BDFSUdfInfo* bd_fs_udf_get_info (const gchar *device, GError **error);
 
+typedef enum {
+    BD_FS_SUPPORT_SET_LABEL = 1 << 1,
+    BD_FS_SUPPORT_SET_UUID = 1 << 2
+} BDFSConfigureFlags;
+
+typedef enum {
+    BD_FS_FSCK_CHECK = 1 << 1,
+    BD_FS_FSCK_REPAIR = 1 << 2
+} BDFSFsckFlags;
+
+typedef enum {
+    BD_FS_FEATURE_OWNERS  = 1 << 1
+} BDFSFeatureFlags;
```

Comment:
I vaguely remember we were talking about flags for xattrs, quota, compression, dedup, multidisk, ... ? Does it fit here?

Reply:
Yes. That's why I added a separate flag here. My idea was `configure` to be for things that can be changed for an existing filesystem and `features` for "features" (don't know how to explain it better). But I agree with @vpodzime there is a thin line between these two. *compression* is a feature but it's also something that can be configured. So `features` should be for things that are not "normal".